src/main/java/com/minecolonies/rankup/Plugininfo.java

```diff
@@ -3,13 +3,13 @@
 /**
  * Contains general information about the plugin.
  */
-public class Plugininfo
+public final class Plugininfo
 {
-    private Plugininfo() {}
-
     public static final String ID          = "rankup";
-    public static final String NAME        = "@name@";
+    public static final String NAME        = "Rankup";
     public static final String VERSION     = "@version@";
     public static final String DESCRIPTION = "@description@";
     public static final String URL         = "http://github.com/Minecolonies/Rankup";
+
+    private Plugininfo() {}
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this left curly brace to the beginning of next line of code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)